src/python/pants/backend/python/lint/bandit/rules_integration_test.py

```diff
@@ -69,6 +71,7 @@ def run_bandit(
             LintResults,
             Params(
                 BanditRequest(BanditFieldSet.create(tgt) for tgt in targets),
+                create_subsystem(LintOptions, reports_dir=reports_dir),  # type: ignore[type-var]
```

Comment:
Rather than doing this, it's better to modify the `args` list for `create_options_bootstrapper` with the CLI arg you want.

You'll need to remove line 31 as well.

--

It would be great to fix Flake8 as well. Sorry I hadn't reviewed that PR - everything looked great except for this issue with the tests.